config.js

```diff
@@ -16,12 +16,20 @@ const ConfigOptions = () => {
     let config = {
         isProduction,
         isDev,
-
         webRootDir: '.',
         assetSrcDir: 'src',
         assetDistDir: 'dist',
         applyRevision: true,
 
+        /**
+         * Fozzie-related variables
+         */
+        fozzieSettings: {
+            useFozziePackageVersion: false,
+            getFozziePackageVersion: consumingPackageConfig && consumingPackageConfig.version,
```

Comment:
Again, I'd keep this generic and call it something like `packageVersion` and put it in the base of the config object.  Can see us needing that in other places potentially in future.

Reply:
Will update this too.